src/redux/books/book.js

```diff
@@ -0,0 +1,60 @@
+export const ADD_BOOK = 'ADD_BOOK';
+export const REMOVE_BOOK = 'REMOVE_BOOK';
+export const EDIT_BOOK = 'EDIT_BOOK';
+
+export const initialState = {
+  books: [
+    {
+      id: 1,
+      title: 'Alarm in the Dark',
+      author: 'Kweku A. Kankam',
+      category: 'Fiction',
+      progress: 80,
+    },
+    {
+      id: 2,
+      title: 'Dark Sun',
+      author: 'Quenia A. Kankam',
+      category: 'Fantasy',
+      progress: 100,
+    },
+    {
+      id: 3,
+      title: 'Oh Joy',
+      author: 'Neil Amponsah',
+      category: 'Romance',
+      progress: 20,
+    },
+  ],
+};
```

Comment:
[REQUIRED] Your initial state should be an empty array not an object with an array that contains items. Kindly fix that. You can refer back to the project instructions.